components/atoms/button/index.stories.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+
+import loginIcon from '_storybook/assets/images/login-icon.svg'
+
+import Button from './index'
+
+storiesOf('Button', module).add('with text', () => (
+  <Button onClick={() => console.info('test')}>Login</Button>
+))
+
+storiesOf('Button', module).add('with icon', () => (
+  <Button iconURL={loginIcon} onClick={() => console.info('test')}>
+    Login
+  </Button>
+))
+
+storiesOf('Button', module).add('is disabled', () => <Button isDisabled>Login</Button>)
```

Comment:
according to the guideline, that doesn't exist yet, we should follow the HTML names.

like: `disabled` instead of `isDisabled`